### DISCUSSION TITLE
2005/07/19/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-2

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#40] [Announcements]
# 2005/07/19/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-2

https://blog.qizhen.xyz/2005/07/19/%E4%BB%A5%E5%BE%80%E8%AE%A1%E6%95%B0%E5%99%A8%E4%B8%8B%E7%9A%84%E7%95%99%E8%A8%80-2

**Comment by ruanqizhen** [comment]
Yoyo
2005年07月22日 11:54 编辑
乖乖,差点就以为全是图片了!:）
